lxd-agent/network_1.10.go

```diff
@@ -0,0 +1,13 @@
+// +build go1.10,!go1.11
```

Comment:
Can we have `lxd/vsock` export both Dial and Listen, then have `lxd-agent` import and use that?

This way, we only have one place to mess with if we want to change implementation.

Reply:
Done